examples/bri-3/docs/zeroKnowledge/circuits/snarkjs.md

```diff
@@ -0,0 +1,44 @@
+# Snarkjs Circuit
+
+This document describes the setup and process for generating zero-knowledge proofs using snarkjs. The circuits are written and compiled using [Circom](https://docs.circom.io/getting-started/installation/). Then, the [snarkjs](https://github.com/iden3/snarkjs) library is used to generate proofs from these compiled circuits.
```

Comment:
Does it make sense to write here a paragraph or two explaining how will this be used in the cotext of BRI3?
For example, ...circuits will be executed during workstep execution to achieve..., where circom will be used to perform the steps .... and snarkjs to performs steps. Outputs of this operation will ...

Reply:
Also, we could add a sentence or two about the reasoning behind choosing this approach for BRI-3, and potential alternatives (i.e. separate piece of software running alongside BRI3 for circuit execution)

Reply:
I will add more details as to how to utilise both circom and snarkjs for writing circuits in upcoming PRs as I update the corresponding code.

I will add a small note in the docs for choosing snarkjs for BRI-3 and potential alternatives.